src/SharedFrameworkValidation/RestoreSDKProject/RestoreSDKProject.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <RuntimeFrameworkVersion>$(_RuntimeFrameworkVersion)</RuntimeFrameworkVersion>
     <CoreFxPackageVersion>4.4.0-$(CoreFxExpectedPrerelease)</CoreFxPackageVersion>
-    <RuntimeIdentifier>$(_PublishRID)</RuntimeIdentifier>
+    <RuntimeIdentifier>$(TargetRid)</RuntimeIdentifier>
```

Comment:
Looks to me like this was previously using the build-on-RID.  @joperezr can confirm if it's OK to change this to the buiding-for-RID, and double check that you aren't missing anything by removing the global property passing that was here before.